cfme/configure/configuration/server_settings.py

```diff
@@ -0,0 +1,416 @@
+        database_synchronization_role = BootstrapSwitch(
+            name="server_roles_database_synchronization")
+        git_owner = BootstrapSwitch(name="server_roles_git_owner")
+        websocket = BootstrapSwitch(name="server_roles_websocket")
+        cockpit_ws = BootstrapSwitch(name="server_roles_cockpit_ws")
+        # STORAGE OPTIONS
+        storage_metrics_processor = BootstrapSwitch(name="server_roles_storage_metrics_processor")
+        storage_metrics_collector = BootstrapSwitch(name="server_roles_storage_metrics_collector")
+        storage_metrics_coordinator = BootstrapSwitch(
+            name="server_roles_storage_metrics_coordinator")
+        storage_inventory = BootstrapSwitch(name="server_roles_storage_inventory")
+        vmdb_storage_bridge = BootstrapSwitch(name="server_roles_vmdb_storage_bridge")
+
+        default_smart_proxy = Text(
+            "//label[contains(text(), 'Default Repository SmartProxy')]/following-sibling::div")
+
+    @View.nested
+    class vmware_console(View):    # noqa
+        """ Class represents Server VWware Console Support Form """
+
+        console_type = BootstrapSelect("console_type")
+
+    @View.nested
+    class ntp_servers(View):    # noqa
+        """ Class represents Server VWware Console Support Form """
+
+        ntp_server_1 = Input(name="ntp_server_1")
+        ntp_server_2 = Input(name="ntp_server_2")
+        ntp_server_3 = Input(name="ntp_server_3")
+
+    @View.nested
+    class smtp_server(View):    # noqa
+        """ Class represents SMTP Server Form """
+
+        host = Input("smtp_host")
+        port = Input("smtp_port")
+        domain = Input("smtp_domain")
+        start_tls = Input("smtp_enable_starttls_auto")
+        ssl_verify = BootstrapSelect("smtp_openssl_verify_mode")
+        auth = BootstrapSelect("smtp_authentication")
+        username = Input("smtp_user_name")
+        password = Input("smtp_password")
+        from_email = Input("smtp_from")
+        to_email = Input("smtp_test_to")
+        verify = Button('Verify')
+
+    @View.nested
+    class web_services(View):   # noqa
+        """ Class represents Server WebServices Form """
+
+        mode = BootstrapSelect(id='webservices_mode')
+        security = BootstrapSelect(id='webservices_security')
+
+    @View.nested
+    class logging_form(View):   # noqa
+        """ Class represents Server Logging Form """
+
+        log_level = BootstrapSelect(id='log_level')
+
+    @View.nested
+    class custom_support_url(View):   # noqa
+        """ Class represents Server Custom Support URL Form """
+
+        url = Input(name='custom_support_url')
+        description = Input(name='custom_support_url_description')
+
+    @property
+    def is_displayed(self):
+        return {
+            self.server.is_active and
+            self.title.text == 'Basic Information'
+        }
+
+
+class ServerInformation(Updateable, Pretty, Navigatable):
```

Comment:
Now we should inherit `NavigatableMixin`, also `appliance` argument is required.

Reply:
done

Reply:
Not here it's not :)